addons/mail/models/mail_activity.py

```diff
@@ -110,6 +110,20 @@ def default_get(self, fields):
             res['res_model_id'] = self.env['ir.model']._get(res['res_model']).id
         return res
 
+    @api.model
+    def _default_activity_type_id(self):
+        ActivityType = self.env["mail.activity.type"]
+        todo = self.env.ref('mail.mail_activity_data_todo', raise_if_not_found=False)
```

Comment:
todo is quite a generic naming, could be activity_type_todo, when reading it is more understandable